server-go/storage.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"github.com/grafeas/grafeas/samples/server/go-server/api"
+	"github.com/grafeas/grafeas/samples/server/go-server/api/server/errors"
+)
+
+// Storager implementations are
```

Comment:
it looks like this comment didn't get finished?

Reply:
updated